source/fundamentals/crud/write-operations/insert.txt

```diff
@@ -14,12 +14,14 @@ Insert
 ------
 
 If you want to add new documents to a collection, you can use
```

Comment:
suggeston: improve tone

I know you didn't make changes to this, but it definitely needs updating I think.

```suggestion
To add new documents to a collection, use
```